invenio_app_ils/schemas/series/series-v1.0.0.json

```diff
@@ -230,6 +242,18 @@
     "title": {
       "type": "string"
     },
+    "updated_by": {
+      "properties": {
+        "type": {
+          "type": "string"
+        },
+        "value": {
+          "type": "string"
+        }
+      },
+      "title": "Last update user information",
```

Comment:
Super minor: if I am not mistaken, it might not be a `user`, but also a script or something else. What about changing the title to something like `Last entity/curator/whatever that updated the record` or similar?